src/weapons/GunController.gd

```diff
@@ -0,0 +1,57 @@
+class_name GunController
+extends Node3D
+
+@export var weapon_aim_speed = 0.1
+
+var b := false
```

Comment:
Var b is meant to be a boolean, but what's it for / can this be more descriptive? 
Same with Var r